backend/core/management/commands/watch_account_creation.py

```diff
@@ -0,0 +1,51 @@
+"""
+Watch and alert when a certain number of accounts are created with the same trusted domaine.
+"""
+from django.core.management.base import BaseCommand
+
+from core.lib.discord.api import write_in_channel
+from core.utils.email_domain import get_user_cnt_per_email_domain
+from core.utils.time import time_ago
+
+
+class Command(BaseCommand):
+    help = "Alert when a certain of accounts are created with the same trusted mail domain."
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "-s",
+            "--since_n_hours",
+            type=int,
+            help="Number of hours ago to watch",
+        )
+
+        # Optional
+        parser.add_argument(
+            "-n",
+            "--n_account",
+            type=int,
+            default=1,
+            help="Number of account to raise alerting (default 1)",
```

Comment:
Not sure that "raise alerting" is correct.

```suggestion
            help="Minimum number of created accounts required to raise an alert.",
```